rust/main/agents/relayer/src/msg/pending_message.rs

```diff
@@ -1060,4 +1063,66 @@ mod test {
 
         assert_eq!(num_retries, expected_retries);
     }
+
+    /// Make sure DEFAULT_MAX_MESSAGE_RETRIES takes around 2 weeks to reach
+    /// so that messages doesn't getting dropped earlier than expected
+    #[test]
+    fn check_default_max_message_retries() {
+        let total_backoff_duration: Duration = (0..DEFAULT_MAX_MESSAGE_RETRIES)
+            .filter_map(|i| {
+                PendingMessage::calculate_msg_backoff(i, DEFAULT_MAX_MESSAGE_RETRIES, None)
+            })
+            .sum();
+
+        eprintln!("Total backoff duration: {:?}", total_backoff_duration);
```

Comment:
nit: intentionally kept or no?

Reply:
I was thinking to keep it, but doesn't really matter so I'll remove it xd